File: GlassDoor/ClientApp/src/app/models/language.ts
import { UserLanguage } from "./user-language";

export class Language {

  constructor(id?: number, userLanguages?: UserLanguage, name?: string)
  {

    this.id = id;
    this.name = name;
    this.userLanguages = userLanguages;
   

  }
  public id: number;
  public name: string;
  public userLanguages: UserLanguage;


}

File: DAL/Models/Language.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class Language
    {
        public int Id { get; set; }
        public string Name{ get; set; }
        public ICollection<UserLanguage> UserLanguages { get; set; }
    }
}
